backend/spec/models/option_spec.rb

```diff
@@ -22,11 +22,17 @@
     end
 
     describe 'when many options are set to correct' do
-      it { expect(option.errors.full_messages).to match(['Single choice questions should have no more than one correct option.']) }
+      it { expect(option.errors.full_messages).to match(['Name has already been taken', 'Single choice questions should have no more than one correct option.']) }
       it { expect(option).not_to be_valid }
     end
   end
 
+  describe 'uniqueness' do
+    before { create(:option) }
```

Comment:
is this necessary?

Reply:
Yes, it's necessary. Because we need to have an option created to test this validation. And in option_spec there is only one let!(build for option).